src/module/protal/views/panduan-view.tsx

```
@@ -0,0 +1,94 @@
+                    </TableCell>
+                  </TableRow>
+                ))}
+              </TableBody>
+            </Table>
+          </div>
+          {/* Card untuk mobile */}
+          <div className="block md:hidden space-y-4">
+            {panduanList.map((panduan: any) => (
+              <div key={panduan.id} className="bg-white rounded-lg shadow p-4 flex flex-col">
+                <div className="font-semibold text-base mb-2">
+                  {panduan.file && panduan.file.url ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)